bitnami/spring-cloud-dataflow/templates/skipper/deployment.yaml

```diff
@@ -80,18 +80,22 @@ spec:
               value: "7577"
             - name: SPRING_CLOUD_CONFIG_ENABLED
               value: "false"
+            - name: SPRING_CLOUD_KUBERNETES_CONFIG_ENABLE_API
+              value: "false"
             - name: SPRING_CLOUD_KUBERNETES_SECRETS_ENABLE_API
-              value: "true"
+              value: "false"
             - name: SPRING_CLOUD_KUBERNETES_SECRETS_PATHS
               value: "/etc/secrets"
-            - name: SPRING_CLOUD_KUBERNETES_CONFIG_NAME
```

Comment:
Could you explain why this should be deleted?

Reply:
Because it's not working actually and it seems not a recommended way to do it (based on documentation also, same link: https://docs.spring.io/spring-cloud-dataflow/docs/current-SNAPSHOT/reference/htmlsingle/#configuration-kubernetes-general).

Also if you compare tests for this chart before my changes and after, you will see this `SPRING_CLOUD_KUBERNETES_CONFIG_NAME` didn't work as expected before, because you will see the app will ignore this config and use H2 database by default instead of MariaDB (which is must be used based on configmap from this chart: https://github.com/bitnami/charts/blob/master/bitnami/spring-cloud-dataflow/templates/server/configmap.yaml#L59).

Before (https://github.com/bitnami/charts/runs/795906002#step:5:893):

<img width="1247" alt="image" src="https://user-images.githubusercontent.com/139017/86003469-6a2b8180-ba44-11ea-9fe6-08c53ef21d58.png">

After (https://github.com/bitnami/charts/pull/2945/checks?check_run_id=814594092#step:5:789):

<img width="1238" alt="image" src="https://user-images.githubusercontent.com/139017/86003614-a7900f00-ba44-11ea-8eac-24732cc75006.png">

Reply:
Tests also fixed: https://github.com/bitnami/charts/runs/819088449

Reply:
@miguelaeh actually with my fixes can use `existing configmap` as well, same as how we use default one, please check these lines of code:
https://github.com/donbeave/bitnami-charts/blob/f55ea309966e3f82f6a53e197ae1426e62e2e423/bitnami/spring-cloud-dataflow/templates/server/deployment.yaml#L198
https://github.com/donbeave/bitnami-charts/blob/f55ea309966e3f82f6a53e197ae1426e62e2e423/bitnami/spring-cloud-dataflow/templates/_helpers.tpl#L113

Actually, it's almost same as before, just use another way to config it, before it was using env (which for some reason doesn't work), that's why I have changed it to the configuration (which recommended by the Spring Cloud Data Flow documentation), which is mount configmap as a file.

The master version (without my changes) gets this error during startup: *No valid primary TaskPlatform configure*.

You can reproduce it by using this flow:

1) Start minikube:
```bash
minikube start --cpus=4 --memory=8192 --kubernetes-version=v1.18.5
```

2) Install `spring-cloud-dataflow` helm chart:
``` bash
helm repo add bitnami https://charts.bitnami.com/bitnami
helm install my-release bitnami/spring-cloud-dataflow
```

3) Check service account was created:
``` bash
~ ❯ kubectl get sa                                                                                                                                                                      х INT at ○ minikube
NAME                               SECRETS   AGE
default                            1         4m9s
my-release-rabbitmq                2         3m49s
my-release-spring-cloud-dataflow   1         3m49s
```

4) Check service account was set in the pod manifest:
```bash
~ ❯ kubectl get po my-release-spring-cloud-dataflow-server-5c87d54f5d-p4qcd -o yaml | grep serviceAccount                                                                                     at ○ minikube
        f:serviceAccount: {}
        f:serviceAccountName: {}
  serviceAccount: my-release-spring-cloud-dataflow
  serviceAccountName: my-release-spring-cloud-dataflow
```

5) Check dataflow server logs:
```bash
~ ❯ kubectl logs -f my-release-spring-cloud-dataflow-server-5c87d54f5d-p4qcd                                                                                                   х INT took 18s at ○ minikube
spring-cloud-dataflow 10:05:51.20
spring-cloud-dataflow 10:05:51.20 Welcome to the Bitnami spring-cloud-dataflow container
spring-cloud-dataflow 10:05:51.20 Subscribe to project updates by watching https://github.com/bitnami/bitnami-docker-spring-cloud-dataflow
spring-cloud-dataflow 10:05:51.20 Submit issues and feature requests at https://github.com/bitnami/bitnami-docker-spring-cloud-dataflow/issues
spring-cloud-dataflow 10:05:51.21
spring-cloud-dataflow 10:05:51.21 INFO  ==> ** Starting Spring Cloud Data Flow setup **
spring-cloud-dataflow 10:05:51.22 INFO  ==> Validating settings in SPRING_CLOUD_DATAFLOW_* env vars
spring-cloud-dataflow 10:05:51.22 WARN  ==> Using Kubernetes Secrets.
spring-cloud-dataflow 10:05:51.22 INFO  ==> ** Spring Cloud Data Flow setup finished! **

spring-cloud-dataflow 10:05:51.22 INFO  ==> ** Starting Spring Cloud Data Flow **
  ____                              ____ _                __
 / ___| _ __  _ __(_)_ __   __ _   / ___| | ___  _   _  __| |
 \___ \| '_ \| '__| | '_ \ / _` | | |   | |/ _ \| | | |/ _` |
  ___) | |_) | |  | | | | | (_| | | |___| | (_) | |_| | (_| |
 |____/| .__/|_|  |_|_| |_|\__, |  \____|_|\___/ \__,_|\__,_|
  ____ |_|    _          __|___/                 __________
 |  _ \  __ _| |_ __ _  |  ___| | _____      __  \ \ \ \ \ \
 | | | |/ _` | __/ _` | | |_  | |/ _ \ \ /\ / /   \ \ \ \ \ \
 | |_| | (_| | || (_| | |  _| | | (_) \ V  V /    / / / / / /
 |____/ \__,_|\__\__,_| |_|   |_|\___/ \_/\_/    /_/_/_/_/_/

Spring Cloud Data Flow Server  (v2.5.2.RELEASE)

2020-07-01 10:05:54.504  INFO 1 --- [           main] b.c.PropertySourceBootstrapConfiguration : Located property source: [BootstrapPropertySource {name='bootstrapProperties-configmap.my-release-spring-cloud-dataflow-server.default'}]
2020-07-01 10:05:54.577  INFO 1 --- [           main] b.c.PropertySourceBootstrapConfiguration : Located property source: [BootstrapPropertySource {name='bootstrapProperties-mariadb-password'}, BootstrapPropertySource {name='bootstrapProperties-mariadb-root-password'}, BootstrapPropertySource {name='bootstrapProperties-secrets.spring-cloud-dataflow-server.default'}]
2020-07-01 10:05:54.632  INFO 1 --- [           main] o.s.c.d.s.s.DataFlowServerApplication    : The following profiles are active: kubernetes
2020-07-01 10:05:56.215  INFO 1 --- [           main] .s.d.r.c.RepositoryConfigurationDelegate : Multiple Spring Data modules found, entering strict repository configuration mode!
2020-07-01 10:05:56.216  INFO 1 --- [           main] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data Map repositories in DEFAULT mode.
2020-07-01 10:05:56.412  INFO 1 --- [           main] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 171ms. Found 1 Map repository interfaces.
2020-07-01 10:05:56.861  INFO 1 --- [           main] .s.d.r.c.RepositoryConfigurationDelegate : Multiple Spring Data modules found, entering strict repository configuration mode!
2020-07-01 10:05:56.863  INFO 1 --- [           main] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data JPA repositories in DEFAULT mode.
2020-07-01 10:05:56.873  INFO 1 --- [           main] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 6ms. Found 0 JPA repository interfaces.
2020-07-01 10:05:57.053  INFO 1 --- [           main] .s.d.r.c.RepositoryConfigurationDelegate : Multiple Spring Data modules found, entering strict repository configuration mode!
2020-07-01 10:05:57.053  INFO 1 --- [           main] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data JPA repositories in DEFAULT mode.
2020-07-01 10:05:57.130  INFO 1 --- [           main] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 76ms. Found 5 JPA repository interfaces.
2020-07-01 10:05:57.306  INFO 1 --- [           main] o.s.cloud.context.scope.GenericScope     : BeanFactory id=6f6f3b1b-d9eb-3a1f-b029-320a2206dc67
2020-07-01 10:05:58.073  INFO 1 --- [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 8080 (http)
2020-07-01 10:05:58.083  INFO 1 --- [           main] o.a.coyote.http11.Http11NioProtocol      : Initializing ProtocolHandler ["http-nio-8080"]
2020-07-01 10:05:58.084  INFO 1 --- [           main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2020-07-01 10:05:58.084  INFO 1 --- [           main] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.34]
2020-07-01 10:05:58.167  INFO 1 --- [           main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2020-07-01 10:05:58.646  INFO 1 --- [           main] o.s.c.d.s.config.web.WebConfiguration    : Start Embedded H2
2020-07-01 10:05:58.647  INFO 1 --- [           main] o.s.c.d.s.config.web.WebConfiguration    : Starting H2 Server with URL: jdbc:h2:tcp://localhost:19092/mem:dataflow
2020-07-01 10:05:59.103  INFO 1 --- [           main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Starting...
2020-07-01 10:05:59.159  INFO 1 --- [           main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Start completed.
2020-07-01 10:05:59.205  INFO 1 --- [           main] o.f.c.internal.license.VersionPrinter    : Flyway Community Edition 6.0.8 by Redgate
2020-07-01 10:05:59.237  INFO 1 --- [           main] o.f.c.internal.database.DatabaseFactory  : Database: jdbc:h2:tcp://localhost:19092/mem:dataflow (H2 1.4)
2020-07-01 10:05:59.254  WARN 1 --- [           main] o.f.c.internal.database.base.Database    : Flyway upgrade recommended: H2 1.4.200 is newer than this version of Flyway and support has not been tested. The latest supported version of H2 is 1.4.199.
2020-07-01 10:05:59.329  INFO 1 --- [           main] o.f.core.internal.command.DbValidate     : Successfully validated 1 migration (execution time 00:00.040s)
2020-07-01 10:05:59.354  INFO 1 --- [           main] o.f.c.i.s.JdbcTableSchemaHistory         : Creating Schema History table "PUBLIC"."flyway_schema_history_dataflow" ...
2020-07-01 10:05:59.403  INFO 1 --- [           main] o.f.core.internal.command.DbMigrate      : Current version of schema "PUBLIC": << Empty Schema >>
2020-07-01 10:05:59.417  INFO 1 --- [           main] o.f.core.internal.command.DbMigrate      : Migrating schema "PUBLIC" to version 1 - INITIAL SETUP
2020-07-01 10:05:59.478  INFO 1 --- [           main] o.f.core.internal.command.DbMigrate      : Successfully applied 1 migration to schema "PUBLIC" (execution time 00:00.087s)
2020-07-01 10:05:59.955  INFO 1 --- [           main] o.hibernate.jpa.internal.util.LogHelper  : HHH000204: Processing PersistenceUnitInfo [name: default]
2020-07-01 10:06:00.720  INFO 1 --- [           main] org.hibernate.Version                    : HHH000412: Hibernate ORM core version 5.4.15.Final
2020-07-01 10:06:01.130  INFO 1 --- [           main] o.hibernate.annotations.common.Version   : HCANN000001: Hibernate Commons Annotations {5.1.0.Final}
2020-07-01 10:06:01.684  INFO 1 --- [           main] org.hibernate.dialect.Dialect            : HHH000400: Using dialect: org.hibernate.dialect.H2Dialect
2020-07-01 10:06:02.444  INFO 1 --- [           main] o.h.validator.internal.util.Version      : HV000001: Hibernate Validator 6.0.19.Final
2020-07-01 10:06:03.533  INFO 1 --- [           main] o.h.e.t.j.p.i.JtaPlatformInitiator       : HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
2020-07-01 10:06:03.559  INFO 1 --- [           main] j.LocalContainerEntityManagerFactoryBean : Initialized JPA EntityManagerFactory for persistence unit 'default'
2020-07-01 10:06:04.097  INFO 1 --- [           main] urationMetadataResolverAutoConfiguration : Final Registry Configurations: {registry-1.docker.io=RegistryConfiguration{registryHost='registry-1.docker.io', user='null', secret='****'', authorizationType=dockeroauth2, manifestMediaType='application/vnd.docker.distribution.manifest.v2+json', disableSslVerification='false', extra={registryAuthUri=https://auth.docker.io/token?service=registry.docker.io&scope=repository:{repository}:pull&offline_token=1&client_id=shell}}}
2020-07-01 10:06:05.006  WARN 1 --- [           main] o.s.c.d.s.c.f.SchedulerConfiguration     : TaskPlatform Kubernetes is selected as primary but has no TaskLaunchers configured
2020-07-01 10:06:05.008  WARN 1 --- [           main] ConfigServletWebServerApplicationContext : Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'org.springframework.cloud.dataflow.server.config.features.TaskConfiguration': Unsatisfied dependency expressed through field 'schedulerService'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'schedulerService' defined in class path resource [org/springframework/cloud/dataflow/server/config/features/SchedulerConfiguration.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.springframework.cloud.dataflow.server.service.SchedulerService]: Factory method 'schedulerService' threw exception; nested exception is java.lang.IllegalStateException: No valid primary TaskPlatform configured
2020-07-01 10:06:05.010  INFO 1 --- [           main] j.LocalContainerEntityManagerFactoryBean : Closing JPA EntityManagerFactory for persistence unit 'default'
2020-07-01 10:06:05.012  INFO 1 --- [           main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Shutdown initiated...
2020-07-01 10:06:05.032  INFO 1 --- [           main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Shutdown completed.
2020-07-01 10:06:05.035  INFO 1 --- [           main] o.apache.catalina.core.StandardService   : Stopping service [Tomcat]
2020-07-01 10:06:05.039  WARN 1 --- [           main] o.a.c.loader.WebappClassLoaderBase       : The web application [ROOT] appears to have started a thread named [H2 TCP Server (tcp://172.17.0.4:19092)] but has failed to stop it. This is very likely to create a memory leak. Stack trace of thread:
 java.net.PlainSocketImpl.socketAccept(Native Method)
 java.net.AbstractPlainSocketImpl.accept(AbstractPlainSocketImpl.java:409)
 java.net.ServerSocket.implAccept(ServerSocket.java:560)
 java.net.ServerSocket.accept(ServerSocket.java:528)
 org.h2.server.TcpServer.listen(TcpServer.java:263)
 org.h2.tools.Server.run(Server.java:610)
 java.lang.Thread.run(Thread.java:748)
2020-07-01 10:06:05.062 ERROR 1 --- [           main] o.s.boot.SpringApplication               : Application run failed

org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'org.springframework.cloud.dataflow.server.config.features.TaskConfiguration': Unsatisfied dependency expressed through field 'schedulerService'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'schedulerService' defined in class path resource [org/springframework/cloud/dataflow/server/config/features/SchedulerConfiguration.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.springframework.cloud.dataflow.server.service.SchedulerService]: Factory method 'schedulerService' threw exception; nested exception is java.lang.IllegalStateException: No valid primary TaskPlatform configured
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:643)
	at org.springframework.beans.factory.annotation.InjectionMetadata.inject(InjectionMetadata.java:130)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.postProcessProperties(AutowiredAnnotationBeanPostProcessor.java:399)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1422)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:594)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:517)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:323)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:226)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:321)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:895)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:878)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:550)
	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.refresh(ServletWebServerApplicationContext.java:141)
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:747)
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:397)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:315)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1226)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1215)
	at org.springframework.cloud.dataflow.server.single.DataFlowServerApplication.main(DataFlowServerApplication.java:50)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:48)
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:87)
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:51)
	at org.springframework.boot.loader.JarLauncher.main(JarLauncher.java:52)
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'schedulerService' defined in class path resource [org/springframework/cloud/dataflow/server/config/features/SchedulerConfiguration.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.springframework.cloud.dataflow.server.service.SchedulerService]: Factory method 'schedulerService' threw exception; nested exception is java.lang.IllegalStateException: No valid primary TaskPlatform configured
	at org.springframework.beans.factory.support.ConstructorResolver.instantiate(ConstructorResolver.java:656)
	at org.springframework.beans.factory.support.ConstructorResolver.instantiateUsingFactoryMethod(ConstructorResolver.java:636)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.instantiateUsingFactoryMethod(AbstractAutowireCapableBeanFactory.java:1338)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:1177)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:557)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:517)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:323)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:226)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:321)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
	at org.springframework.beans.factory.config.DependencyDescriptor.resolveCandidate(DependencyDescriptor.java:276)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:1306)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:1226)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:640)
	... 27 common frames omitted
Caused by: org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.springframework.cloud.dataflow.server.service.SchedulerService]: Factory method 'schedulerService' threw exception; nested exception is java.lang.IllegalStateException: No valid primary TaskPlatform configured
	at org.springframework.beans.factory.support.SimpleInstantiationStrategy.instantiate(SimpleInstantiationStrategy.java:185)
	at org.springframework.beans.factory.support.ConstructorResolver.instantiate(ConstructorResolver.java:651)
	... 40 common frames omitted
Caused by: java.lang.IllegalStateException: No valid primary TaskPlatform configured
	at org.springframework.cloud.dataflow.server.config.features.SchedulerConfiguration.primaryTaskPlatform(SchedulerConfiguration.java:110)
	at org.springframework.cloud.dataflow.server.config.features.SchedulerConfiguration.schedulerService(SchedulerConfiguration.java:77)
	at org.springframework.cloud.dataflow.server.config.features.SchedulerConfiguration$$EnhancerBySpringCGLIB$$35faa25e.CGLIB$schedulerService$0(<generated>)
	at org.springframework.cloud.dataflow.server.config.features.SchedulerConfiguration$$EnhancerBySpringCGLIB$$35faa25e$$FastClassBySpringCGLIB$$91a154e2.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invokeSuper(MethodProxy.java:244)
	at org.springframework.context.annotation.ConfigurationClassEnhancer$BeanMethodInterceptor.intercept(ConfigurationClassEnhancer.java:331)
	at org.springframework.cloud.dataflow.server.config.features.SchedulerConfiguration$$EnhancerBySpringCGLIB$$35faa25e.schedulerService(<generated>)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.springframework.beans.factory.support.SimpleInstantiationStrategy.instantiate(SimpleInstantiationStrategy.java:154)
	... 41 common frames omitted
```

As you can see from this log, it's using the H2 database instead of MariaDB, because that config didn't apply for some reason.

Reply:
Hi @donbeave,
Thank you for sharing those steps and for the clarification!

Reply:
@miguelaeh you are welcome!